README.rst

```diff
@@ -142,6 +142,16 @@ You should now be able to browse to a ``Genes`` page and search for
 ``CDC20`` or ``ENSG00000117399``.
 
 
+External URL Favicons
+^^^^^^^^^^^^^^^^^^^^^
+
+Mapr can automatically convert external URLs in mapr categories to clickable favicons .
+OMERO.web must be configured with the Django redis cache https://docs.openmicroscopy.org/omero/5/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.html?highlight=redis#configuring-omero-web
+
+To use this feature the mapr category key-value pair such as `Gene Identifier` must be followed by a key called `Gene Identifier URL`.
+The `Gene Identifier URL` key-value pair will be hidden.
```

Comment:
Why is redis a requirement for URLs, and how are the Favicons specified?
Maybe a bit more detail might be nice. e.g. A favicon link to the URL will be added beside the value of the `Gene Identifier` pair.